packages/ui/lit-grid/src/dx-grid.ts

```diff
@@ -1228,19 +1228,52 @@ export class DxGrid extends LitElement {
 
     const cellReadonly = this.cell(col, row, plane)?.readonly;
     if (cellReadonly !== undefined) {
-      return cellReadonly;
+      // Convert the possible string values to boolean
+      return cellReadonly === 'no-text-select' || cellReadonly === 'text-select' || cellReadonly === true;
```

Comment:
Strings will be `true`, so maybe just `!!cellReadonly`?

Reply:
That's a little magical for me.

Reply:
Fair enough! This is still fine.